larky/src/test/resources/quick_tests/test_packager.star

```diff
@@ -0,0 +1,26 @@
+load("@stdlib//unittest", "unittest")
+load("@vendor//asserts", "asserts")
+load("@vendor//ISO8583/Parser/Parser", Parser="Parser")
```

Comment:
is `ISO8583` a vendored package? Can I go `pip install ${python_package}` and I will be able to have a local ISO8583 API to play in a normal python environment?

Reply:
pyiso8583 uses JSON packager, jpos packager has XML format. jpos is used by the current TCP proxy implementation. atm not sure what is a preferable way of the packager representaion.